src/dataLoaders/components/collectorsWizard/TelegrafUIRefreshWizard.tsx

```diff
@@ -0,0 +1,178 @@
+// Libraries
+import React, {PureComponent} from 'react'
+import Loadable from 'react-loadable'
+import {connect, ConnectedProps} from 'react-redux'
+import {withRouter, RouteComponentProps} from 'react-router-dom'
+
+// Components
+import {ErrorHandling} from 'src/shared/decorators/errors'
+import {Overlay} from '@influxdata/clockface'
+import {PluginCreateConfigurationFooter} from 'src/writeData/components/PluginCreateConfigurationFooter'
+
+const spinner = <div />
```

Comment:
should this be an actual loading spinner?

Reply:
Good question. This was carried over from another Wizard that this new Wizard is based on. The old Wizard clearly has some tech debt regarding the spinner.

I have put in a `<PageSpinner />` rather than a `<div>` for the new Wizard. Looks much better.

Thanks for the sharp observation!